rook/workflows/iperf.yml

```diff
@@ -0,0 +1,37 @@
+options:
+  env:
+    KUBECONFIG: ./kubeconfig/config
+
+steps:
+- id: run
+  uses: docker://bitnami/kubectl:1.17.4
+  runs: [bash]
```

Comment:
minor cosmetic comment: if the step is only running one command, then it can be expressed in the `runs:` attribute. So

```yaml
- id: run
  uses: docker://bitnami/kubectl:1.17.4
  args: [create, -n, kubestone, -f, ./kubestone-iperf/iperf.yaml]
```

Reply:
Ahh ! Thats really cool. Thanks